{"@t":"2019-04-21T04:55:55.4749956Z","@mt":"{StartMessage} [Timing {TimingId}]","StartMessage":"Booting Umbraco 8.0.1 on FCD4A1C04EF9.","TimingId":"6b7e985","SourceContext":"Umbraco.Core.Runtime.CoreRuntime","ProcessId":1868,"ProcessName":"w3wp","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC1ROOT","MachineName":"FCD4A1C04EF9","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"986192c2-eeae-425f-88d7-8880f094c1ed"}
{"@t":"2019-04-21T04:55:56.0218708Z","@mt":"Acquiring.","SourceContext":"Umbraco.Core.MainDom","ProcessId":1868,"ProcessName":"w3wp","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC1ROOT","MachineName":"FCD4A1C04EF9","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"986192c2-eeae-425f-88d7-8880f094c1ed"}
{"@t":"2019-04-21T04:55:56.0218708Z","@mt":"Acquired.","SourceContext":"Umbraco.Core.MainDom","ProcessId":1868,"ProcessName":"w3wp","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC1ROOT","MachineName":"FCD4A1C04EF9","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"986192c2-eeae-425f-88d7-8880f094c1ed"}
{"@t":"2019-04-21T04:56:28.2648311Z","@mt":"{FailMessage} ({Duration}ms) [Timing {TimingId}]","@l":"Error","@x":"Umbraco.Core.Exceptions.BootFailedException: A connection string is configured but Umbraco could not connect to the database.\r\n   at Umbraco.Core.RuntimeState.DetermineRuntimeLevel(IUmbracoDatabaseFactory databaseFactory, ILogger logger)\r\n   at Umbraco.Core.Runtime.CoreRuntime.DetermineRuntimeLevel(IUmbracoDatabaseFactory databaseFactory, IProfilingLogger profilingLogger)\r\n   at Umbraco.Core.Runtime.CoreRuntime.Boot(IRegister register, DisposableTimer timer)","FailMessage":"Boot failed.","Duration":32814,"TimingId":"6b7e985","SourceContext":"Umbraco.Core.Runtime.CoreRuntime","ProcessId":1868,"ProcessName":"w3wp","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC1ROOT","MachineName":"FCD4A1C04EF9","Log4NetLevel":"ERROR","HttpRequestNumber":1,"HttpRequestId":"986192c2-eeae-425f-88d7-8880f094c1ed"}
{"@t":"2019-04-21T04:56:38.4089535Z","@mt":"Stopping ({SignalSource})","SignalSource":"environment","SourceContext":"Umbraco.Core.MainDom","ProcessId":1868,"ProcessName":"w3wp","ThreadId":10,"AppDomainId":2,"AppDomainAppId":"LMW3SVC1ROOT","MachineName":"FCD4A1C04EF9","Log4NetLevel":"INFO "}
{"@t":"2019-04-21T04:56:38.4089535Z","@mt":"Released ({SignalSource})","SignalSource":"environment","SourceContext":"Umbraco.Core.MainDom","ProcessId":1868,"ProcessName":"w3wp","ThreadId":10,"AppDomainId":2,"AppDomainAppId":"LMW3SVC1ROOT","MachineName":"FCD4A1C04EF9","Log4NetLevel":"INFO "}
{"@t":"2019-04-21T04:58:49.8968664Z","@mt":"{StartMessage} [Timing {TimingId}]","StartMessage":"Booting Umbraco 8.0.1 on FCD4A1C04EF9.","TimingId":"fbcbb9d","SourceContext":"Umbraco.Core.Runtime.CoreRuntime","ProcessId":1868,"ProcessName":"w3wp","ThreadId":1,"AppDomainId":3,"AppDomainAppId":"LMW3SVC1ROOT","MachineName":"FCD4A1C04EF9","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"e8fe6920-8c6a-424d-86fa-5a17e7147a67"}
{"@t":"2019-04-21T04:58:50.2562335Z","@mt":"Acquiring.","SourceContext":"Umbraco.Core.MainDom","ProcessId":1868,"ProcessName":"w3wp","ThreadId":1,"AppDomainId":3,"AppDomainAppId":"LMW3SVC1ROOT","MachineName":"FCD4A1C04EF9","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"e8fe6920-8c6a-424d-86fa-5a17e7147a67"}
{"@t":"2019-04-21T04:58:50.2562335Z","@mt":"Acquired.","SourceContext":"Umbraco.Core.MainDom","ProcessId":1868,"ProcessName":"w3wp","ThreadId":1,"AppDomainId":3,"AppDomainAppId":"LMW3SVC1ROOT","MachineName":"FCD4A1C04EF9","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"e8fe6920-8c6a-424d-86fa-5a17e7147a67"}
{"@t":"2019-04-21T04:59:17.2282521Z","@mt":"{FailMessage} ({Duration}ms) [Timing {TimingId}]","@l":"Error","@x":"Umbraco.Core.Exceptions.BootFailedException: A connection string is configured but Umbraco could not connect to the database.\r\n   at Umbraco.Core.RuntimeState.DetermineRuntimeLevel(IUmbracoDatabaseFactory databaseFactory, ILogger logger)\r\n   at Umbraco.Core.Runtime.CoreRuntime.DetermineRuntimeLevel(IUmbracoDatabaseFactory databaseFactory, IProfilingLogger profilingLogger)\r\n   at Umbraco.Core.Runtime.CoreRuntime.Boot(IRegister register, DisposableTimer timer)","FailMessage":"Boot failed.","Duration":27355,"TimingId":"fbcbb9d","SourceContext":"Umbraco.Core.Runtime.CoreRuntime","ProcessId":1868,"ProcessName":"w3wp","ThreadId":1,"AppDomainId":3,"AppDomainAppId":"LMW3SVC1ROOT","MachineName":"FCD4A1C04EF9","Log4NetLevel":"ERROR","HttpRequestNumber":1,"HttpRequestId":"e8fe6920-8c6a-424d-86fa-5a17e7147a67"}
{"@t":"2019-04-21T04:59:27.2908531Z","@mt":"Stopping ({SignalSource})","SignalSource":"environment","SourceContext":"Umbraco.Core.MainDom","ProcessId":1868,"ProcessName":"w3wp","ThreadId":10,"AppDomainId":3,"AppDomainAppId":"LMW3SVC1ROOT","MachineName":"FCD4A1C04EF9","Log4NetLevel":"INFO "}
{"@t":"2019-04-21T04:59:27.2908531Z","@mt":"Released ({SignalSource})","SignalSource":"environment","SourceContext":"Umbraco.Core.MainDom","ProcessId":1868,"ProcessName":"w3wp","ThreadId":10,"AppDomainId":3,"AppDomainAppId":"LMW3SVC1ROOT","MachineName":"FCD4A1C04EF9","Log4NetLevel":"INFO "}
